setup.py

```diff
@@ -56,6 +60,7 @@
         'ipykernel',
         'flatten_json',
         'cryptography>=3.4.6',
+        'cloudpickle',
```

Comment:
`dill` should probably be added here as well

Reply:
I do not think we want to include all plugins dependencies into requirements.